tools/set_repo_labels.py

```diff
@@ -28,6 +28,7 @@
     'work in progress': '0052cc',
     'wontfix': 'ffffff',
     'migration': 'd4c5f9',
+    'ready to merge': 'bfdadc',
```

Comment:
What is the difference with `approved`?
Do we have a document with the suggested usage of each label?

Reply:
@elicoidal approved is fine with me. I had not noticed it.

I actually used ready to merge from #346, as a first PR to https://github.com/OCA/oca-github-bot/pull/1

Reply:
@elicoidal after further thinking, I think both are useful: approved when all approvals are ok. Then ready to merge when other conditions (like the PR age are met). I updated https://github.com/OCA/oca-github-bot/pull/1 with this concept.